index.html

```diff
@@ -14,15 +15,15 @@
       <h2>The Main Content</h2>
       <div class='left'>
 	This is the left side
-	<img src='https://jesse.sh/img/me.jpg'>
+	<img src='https://jesse.sh/img/me.jpg' alt='Man on an electric guitar'>
       </div>
       <div class='right'>
 	This is the right side
       </div>
     </div>
-    <aside>
+    <nav>
```

Comment:
👍  `aside` is already semantic but will count this. Generally talking about `div`s and `span`s when referring to non-semantic names